src/backend/app/Services/AbstractExportService.php

```diff
@@ -117,4 +117,20 @@ public function saveSpreadSheet(): string {
             throw new SpreadSheetNotSavedException($e->getMessage());
         }
     }
+
+    public function saveCsv(): string {
+        try {
+            $uuid = Str::uuid()->toString();
+            $fileName = storage_path('appliance').'/'.$this->getPrefix().'-'.$uuid.'.csv';
+            $this->setRecentlyCreatedSpreadSheetId($uuid);
+
+            // Create CSV writer
+            $writer = IOFactory::createWriter($this->spreadsheet, 'Csv');
+            $writer->save($fileName);
+
+            return $fileName;
+        } catch (\Exception $e) {
+            throw new SpreadSheetNotSavedException($e->getMessage());
+        }
+    }
```

Comment:
Do I understand correctly, to create a simple CSV file, we are 

- Creating a spreadsheet by reading a xlsx template
- Transforming our data and inserting it into specific cell in the xlsx
- Convert xlsx to csv

Is this really the way to do it 😅 ? I wonder if there is a more simple way in PHP to achieve this.

Reply:
😁 yeah I will find a simpler approach to this. Thanks